pot/address.go

```diff
@@ -75,6 +77,49 @@ func (a Address) Bytes() []byte {
 	return a[:]
 }
 
+// Distance returns the distance between address a and address b as a big integer using the distance metric defined in the swarm specfication
+// Fails if not all addresses are of equal length
+func Distance(x, y interface{}) (*big.Int, error) { // a Address, b Address) (*big.Int, error){
```

Comment:
Are interface{} arguments really needed here. They are evaluated only by ToBytes function which does an expensive type assertion to return []byte. Why not just accept []byte?